pom.xml

```diff
@@ -49,13 +44,29 @@
             </exclusions>
         </dependency>
 
-        <!-- https://mvnrepository.com/artifact/io.springfox/springfox-boot-starter -->
         <dependency>
             <groupId>io.springfox</groupId>
             <artifactId>springfox-boot-starter</artifactId>
             <version>${springfox.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.postgresql</groupId>
+            <artifactId>postgresql</artifactId>
+            <version>42.2.19</version>
```

Comment:
Suggestion to keep all versions in one place - properties. Please, move it. The same for `liquibase-core`

Reply:
done